.github/workflows/examples-nxp.yaml

```diff
@@ -290,12 +290,13 @@ jobs:
               with:
                 platform: nxp
 
-            - name: Build RW61x Zephyr examples
+            - name: Build NXP Zephyr examples
               run: |
                   scripts/run_in_build_env.sh "\
                       ./scripts/build/build_examples.py \
                       --target nxp-rw61x-zephyr-all-clusters \
                       --target nxp-rw61x-zephyr-thermostat \
                       --target nxp-rw61x-zephyr-laundry-washer-factory \
+                      --target nxp-rw61x_eth-zephyr-thermostat-ethernet \
```

Comment:
Is this a typo `rw61x_eth`?

Reply:
it's not, I added a new board `rw61x_eth` because to get ethernet support, you need to make hardware modifications. So, in zephyr, we see it as a board variant. I wanted to reflect this here.

Reply:
Nevermind, I saw the changes below.